bundle/README.md

```diff
@@ -152,3 +152,111 @@ To configure `bookinfo` with a gateway using `Gateway API`:
     $ echo "http://${GATEWAY_URL}/productpage"
     ```
 
+## Istio Addons Integrations
+
+Istio can be integrated with other software to provide additional functionality (More information can be found in: https://istio.io/latest/docs/ops/integrations/). The Sail Operator provides support for the following integrations:
+
+### Prometheus
+
+`Prometheus` is an open-source systems monitoring and alerting toolkit. You can use `Prometheus` with the Sail Operator to keep an eye on how healthy Istio and the apps in the service mesh are, for more information, see [Prometheus](https://istio.io/latest/docs/ops/integrations/prometheus/). Take into count that you will need to have installed the istio control plane before to start with this integration.
```

Comment:
For the last sentence, I think you mean "Take into account that..." (For this and the other paragraphs). That said, I think it's pretty clear that you'd need Istio to be installed for the Istio addons to work, so it might not be necessary to state this.